build.gradle

```diff
@@ -165,14 +164,16 @@ String checkForEapWithBuildNumberHigherThan(String latestEapBuildNumber) {
 }
 
 task updateEapBuildNumber {
-  Properties properties = project.intellijVersionsProperties()
-  File propFile = project.intellijVersionsFile()
-  String latestEapBuildNumber = project.intellijVersions.eapOfLatestSupportedMajor
-  String newerEapBuildNumber = latestEapBuildNumber ? project.checkForEapWithBuildNumberHigherThan(latestEapBuildNumber.split('-')[0]) :
-          project.checkForEapWithBuildNumberHigherThan(intellijVersionToBuildNumber(project.intellijVersions.latestStable)+'.999999.999999')
-  if (newerEapBuildNumber) {
-    properties.setProperty("eapOfLatestSupportedMajor", newerEapBuildNumber+"-EAP-SNAPSHOT")
-    properties.store(propFile.newWriter(), /*comments*/ null)
+  doLast {
```

Comment:
😯 wow so weird... so task body without `doLast` is auto-executed?

Reply:
yes - everything outside doLast is auto-executed during configuration phase (so when .`/gradlew `is run)